chain/client/src/view_client_actor.rs

```diff
@@ -536,6 +561,29 @@ impl ViewClientActorInner {
         })
     }
 
+    /// Read the receipts that are outgoing from the given execution outcome,
+    /// even before they have been validated on chain and stored in
+    /// DbCol::Receipts.
+    fn outgoing_receipts_for_outcome(
+        &self,
+        outcome: &ExecutionOutcomeWithIdView,
+    ) -> Result<Vec<Receipt>, TxStatusError> {
+        let epoch_id = &self.epoch_manager.get_epoch_id(&outcome.block_hash).into_chain_error()?;
+        let shard_id = account_id_to_shard_id(
+            self.epoch_manager.as_ref(),
+            &outcome.outcome.executor_id,
+            epoch_id,
+        )
+        .into_chain_error()?;
+        let block = self.chain.get_block(&outcome.block_hash)?;
+        let outgoing_receipts = self.chain.get_outgoing_receipts_for_shard(
```

Comment:
`get_outgoing_receipts_for_shard` isn't what we want here, use just `get_outgoing_receipts` instead

Reply:
ah, yes that's easier, I didn't see that function on ChainStoreAccess

Thanks for pointing it out, I fixed it now